v2.0/getting-started/rkt/vagrant/index.md

```diff
@@ -57,24 +58,73 @@ help.  Remember to shut down the VMs with `vagrant halt` before you reboot.
 
 You should also verify each host can access etcd.  The following will return an error if etcd is not available.
 
-    curl -L http://localhost:2379/version
+    curl -L http://172.18.18.101:2379/version
+
+And finally check that `rkt` is running on both hosts by running
+
+    sudo rkt list
+
+## 2. Try out Calico Networking
+
+Now that you have a basic two node CoreOS cluster setup, see the [Calico networking with rkt]({{site.baseurl}}/{{page.version}}/getting-started/rkt/tutorials/basic)
+
+[virtualbox]: https://www.virtualbox.org/
+[vagrant]: https://www.vagrantup.com/downloads.html
+[git]: http://git-scm.com/
+
+---
+title: Calico networking with rkt
+---
+
+
+This tutorial describes how to set up a Calico cluster in a pure rkt environment
+using CNI with Calico specific network and IPAM drivers.
+
+Using the Calico CNI plugin, the required network setup and configuration
+for networking containers using Calico is handled automatically as part of the
+standard network and container lifecycle.  Provided the network is created
+using the Calico driver, creating a container using that network will
+automatically add the container to the Calico network, creating all necessary
+Calico configuration and setting up the interface and routes in the container
+accordingly.
+
+The Calico IPAM driver must be used in addition to the the Calico network
+driver.  This provides IP address management using the configured Calico IP
+Pools as address pools for the container, preferentially selecting sub-blocks
+of IPs for a particular host.
+
+## 1. Environment setup
+
+To run through the worked example in this tutorial you will need to set up two hosts
+with a number of installation dependencies.
 
+Follow the instructions in the tutorial below to set up a virtualized
+environment using Vagrant - be sure to follow the appropriate instructions
+for _Running the Calico rkt tutorials on CoreOS using Vagrant and VirtualBox_.
+
+- [Vagrant install with CoreOS]({{site.baseurl}}/{{page.version}}/getting-started/rkt/installation/vagrant-coreos/)
+
+
+If you have everything set up properly you should have `calicoctl` in your
+`$PATH`, and two hosts called `calico-01` and `calico-02`.
 
 ## 2. Starting Calico services
 
 Once you have your cluster up and running, start Calico on both hosts
 
-    sudo calicoctl node  --runtime=rkt
+```shell
+sudo rkt run --stage1-path=/usr/share/rkt/stage1-fly.aci --set-env=ETCD_ENDPOINTS=http://172.18.18.101:2379 --insecure-options=image --volume=birdctl,kind=host,source=/var/run/calico,readOnly=false --mount volume=birdctl,target=/var/run/calico --volume=mods,kind=host,source=/lib/modules,readOnly=false  --mount volume=mods,target=/lib/modules --volume=logs,kind=host,source=/var/log/calico,readOnly=false --mount volume=logs,target=/var/log/calico --set-env=IP=autodetect --net=host quay.io/calico/node:v1.0.0-rc1 &
```

Comment:
Can you make this easier to ready by adding an escaped newline between each flag like so:
```
sudo rkt run --stage1-path=/usr/share/rkt/stage1-fly.aci \
--set-env=ETCD_ENDPOINTS=http://172.18.18.101:2379 \
--insecure-options=image \
--volume=birdctl,kind=host,source=/var/run/calico,readOnly=false ...
```